### PR TITLE
Update cooks

### PR DESCRIPTION
**Pull request by Laursen79**

### CODE REVIEW

File: cooks [hunk 1]
```diff
@@ -1,3 +1,4 @@
 28/3 - 23: Magnus
 4/4  - 23: Simon & Joms
 18/4 - 23: Alex
+1/5  - 23: Lohse
```
Comment (t-lohse) [inline cooks]:
```suggestion
2/5 - 23: Lohse
```

**Review by t-lohse — CHANGES_REQUESTED** [review]

**Review by alexsteffensen — APPROVED** [review]
LGTM

**Comment by JE-FH** [comment]
review was not good enugh

**Review by alexsteffensen — APPROVED** [review]

File: cooks [hunk 2]
```diff
@@ -1,3 +1,4 @@
 28/3 - 23: Magnus
 4/4  - 23: Simon & Joms
```
Comment (t-lohse) [inline cooks:2]:
```suggestion
4/4 - 23: Simon & Joms
```

**Review by t-lohse — CHANGES_REQUESTED** [review]

**Review by KiraStaehr — APPROVED** [review]

File: cooks [hunk 3]
```diff
@@ -1,3 +1,4 @@
 28/3 - 23: Magnus
-4/4  - 23: Simon & Joms
+4/4 - 23: Simon & Joms
```
Comment (t-lohse) [inline cooks]:
```suggestion
4/4  - 23: Simon & Joms
```

File: cooks [hunk 4]
```diff
@@ -1,3 +1,4 @@
...
 18/4 - 23: Alex
+2/5 - 23: Lohse
```
Comment (t-lohse) [inline cooks]:
```suggestion
2/5  - 23: Lohse
```

**Review by t-lohse — CHANGES_REQUESTED** [review]
Pls also add what dish was served each day

**Comment by Laursen79** [comment]
> Pls also add what dish was served each day

I think this would be a bit much, but anyone is welcome to do it. I have created a seperate issue for this.

**Review by t-lohse — APPROVED** [review]
Looks good to DEEZE NUTS

**Pull request merged**